### ISSUE TITLE
Bug: libva loading error

### ISSUE THREAD

**Issue by yarons** [ferdium/ferdium-app#350]
### Avoid duplicates

- [X] I have searched the [issues tracker](https://github.com/ferdium/ferdium-app/issues) for a bug report similar to mine, in vain

### Ferdium Version

6.0.0-nightly.75

### What Operating System are you using?

Other Linux

### Operating System Version

Manjaro 21.3.0 Ruah (fully upgraded)

### What arch are you using?

x64

### Expected Behavior

libva should load to improve graphics performance.

### Actual Behavior

libva is unable to load so graphics performance might be inferior.

### Steps to reproduce

1. Start Ferdium from a console.

### Additional information

libva error: /usr/lib/dri/i965_drv_video.so init failed
Probably occurs only with Intel GPUs.

Additional details on this error:
https://github.com/intel/libva/issues/375
https://askubuntu.com/questions/1093954/vainfo-returns-libva-error-usr-lib-x86-64-linux-gnu-dri-i965-drv-video-so-ini
https://gitlab.freedesktop.org/gstreamer/gstreamer-vaapi/-/issues/278
https://www.reddit.com/r/linuxquestions/comments/rkn3j5/chromium_libva_error_usrlibdrii965_drv_videoso/
https://bbs.archlinux.org/viewtopic.php?id=233143
etc.

**Comment by etrnal70** [comment]
I don't have this error. What's your system configuration ? AFAIK modern intel gpu should use `intel-media-driver` instead of intel libva driver

**Comment by yarons** [comment]
```
$ sudo inxi -G                                                                                                                                                                                        
Graphics:
  Device-1: Intel CometLake-U GT2 [UHD Graphics] driver: i915 v: kernel
  Device-2: IMC Networks Integrated Camera type: USB driver: uvcvideo
  Display: x11 server: X.Org v: 21.1.3 with: Xwayland v: 22.1.2 driver: X:
    loaded: modesetting gpu: i915 resolution: 1: 1920x1080~60Hz
    2: 1920x1080~60Hz 3: 1920x1080
  OpenGL: renderer: Mesa Intel UHD Graphics (CML GT2) v: 4.6 Mesa 22.1.2
```

How can I tell which I'm using?

**Comment by etrnal70** [comment]
What's your `echo $LIBVA_DRIVER_NAME` or `vainfo` output ?

**Comment by etrnal70** [comment]
Try to install `intel-media-driver` and set `LIBVA_DRIVER_NAME=iHD` in your environment variables

**Comment by yarons** [comment]
Thanks @etrnal70 It's solved on my machine but what can we do to make sure the other users won't run into it?

**Comment by etrnal70** [comment]
I think it's quite different scope since it's Linux-only. But i think if you use manjaro default configuration, you can report to the dev to change the default suitable intel driver for modern device

**Comment by yarons** [comment]
I will, thanks.

**Comment by cino** [comment]
Closing this as you've resolved this for your machine and we have not had others who encountered this issue. If someone has this issue please open up a new issue.

**Comment by luisaureliocasoni** [comment]
Hello, I have got the same error on the recently Ferdium Flatpak version on my Manjaro computer.

Specifying the flag `LIBVA_DRIVER_NAME=iHD` on the environment - I used the `env` command to pass the flag and run Ferdium has solved my problem.

This comment is only for purpose to document same error as of the ticket, and document that this flag `LIBVA_DRIVER_NAME=iHD` solves my problem.

Thanks for your attention.

```
Version: 6.7.0
Server: No Server
Electron: 27.1.3
Chrome: 118.0.5993.159
Node.js: 18.17.1
Platform: Linux 6.5
Arch: x64
Build date: Fri Dec 22 2023 15:26:57 GMT-0400 (Horário Padrão do Amazonas)
Git SHA: 2f449ce7
Git branch: release
```
